source/simulator/parameters.cc

```diff
@@ -773,15 +787,17 @@ namespace aspect
                              "did not detect its presence when you called 'cmake'."));
 #endif
 
-    surface_pressure              = prm.get_double ("Surface pressure");
-    adiabatic_surface_temperature = prm.get_double ("Adiabatic surface temperature");
-    pressure_normalization        = prm.get("Pressure normalization");
-
-    use_direct_stokes_solver      = prm.get_bool("Use direct solver for Stokes system");
-    linear_stokes_solver_tolerance= prm.get_double ("Linear solver tolerance");
-    n_cheap_stokes_solver_steps   = prm.get_integer ("Number of cheap Stokes solver steps");
-    temperature_solver_tolerance  = prm.get_double ("Temperature solver tolerance");
-    composition_solver_tolerance  = prm.get_double ("Composition solver tolerance");
+    surface_pressure               = prm.get_double ("Surface pressure");
+    adiabatic_surface_temperature  = prm.get_double ("Adiabatic surface temperature");
+    pressure_normalization         = prm.get("Pressure normalization");
+
+    use_direct_stokes_solver       = prm.get_bool("Use direct solver for Stokes system");
+    linear_stokes_solver_tolerance = prm.get_double ("Linear solver tolerance");
+    linear_solver_A_block_tolerance= prm.get_double ("Linear solver A block tolerance");
+    linear_solver_S_block_tolerance= prm.get_double ("Linear solver S block tolerance");
+    n_cheap_stokes_solver_steps    = prm.get_integer ("Number of cheap Stokes solver steps");
+    temperature_solver_tolerance   = prm.get_double ("Temperature solver tolerance");
+    composition_solver_tolerance   = prm.get_double ("Composition solver tolerance");
```

Comment:
If you align all of these parameters, please add another space, so there is at least one space between parameter and equal sign: `linear_solver_A_block_tolerance =`. Not your fault, the old version looked strange already. Or did astyle do this?